openreview/arr/helpers.py

```diff
@@ -2101,3 +2111,133 @@ def get_resubmissions(submissions, previous_url_field):
         lambda s: previous_url_field in s.content and 'value' in s.content[previous_url_field] and len(s.content[previous_url_field]['value']) > 0, 
         submissions
     ))
+
+def setup_arr_root_groups(client, support_user):
+    """
+    Creates the root ARR groups and meta-invitation.
+    Domain: aclweb.org/ACL/ARR
+    Roles: Reviewers, Area_Chairs, Senior_Area_Chairs, Ethics_Reviewers, Ethics_Chairs, Editors_In_Chief
+    
+    This function:
+    1. Ensures the domain group aclweb.org/ACL/ARR exists
+    2. Creates the meta invitation aclweb.org/ACL/ARR/-/Edit
+    3. Creates role groups (e.g. aclweb.org/ACL/ARR/Reviewers)
+    """
+    # TODO: Add pointer to the current cycle for edge posting
+    domain = 'aclweb.org/ACL/ARR'
+    roles = ARR_ROLE_NAMES
+    super_meta_invitation_id = support_user.split('/')[0] + '/-/Edit'
+    
+    # 1. Build domain group path (aclweb.org, aclweb.org/ACL, aclweb.org/ACL/ARR)
+    path_components = domain.split('/')
+    paths = ['/'.join(path_components[0:index+1]) for index, path in enumerate(path_components)]
+    
+    for p in paths:
+        group = openreview.tools.get_group(client, id=p)
+        if group is None:
+            client.post_group_edit(
+                invitation=super_meta_invitation_id,
+                readers=['everyone'],
+                writers=['~Super_User1'],
+                signatures=['~Super_User1'],
+                group=openreview.api.Group(
+                    id=p,
+                    readers=['everyone'],
+                    nonreaders=[],
+                    writers=[p],
+                    signatories=[p],
+                    signatures=['~Super_User1'],
+                    members=[],
+                    details={'writable': True}
+                )
+            )
+    
+    # 2. Create meta invitation (aclweb.org/ACL/ARR/-/Edit)
+    meta_invitation_id = f'{domain}/-/Edit'
+    meta_invitation = openreview.tools.get_invitation(client, meta_invitation_id)
+    
+    if meta_invitation is None:
+        # Get process content from venue module
+        process_dir = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'venue', 'process')
+        invitation_edit_process_path = os.path.join(process_dir, 'invitation_edit_process.py')
```

Comment:
is there a reason to add these process functions?

Reply:
No I can remove this, I think it was there when I was testing out making different invitations but none of the root invitations should need to do the propagation of changes